includes/functions/functions_lookups.php

```diff
@@ -336,7 +336,7 @@ function zen_requires_attribute_selection($products_id) {
               left join " . TABLE_PRODUCTS_OPTIONS . " po on pa.options_id = po.products_options_id
               where pa.products_id = " . (int)$products_id . "
               and po.language_id = " . (int)$_SESSION['languages_id'] . "
-              group by pa.options_id, pa.products_attributes_id";
+              group by pa.options_id";
```

Comment:
Just checking: Which MySQL version did you test this against? 

I added the extra `group by` field as part of #662 via https://github.com/scottcwilson/zc-v1-series/commit/1399f1ebcaea4f3f5dfb874c45a86be59f08e897